programs/nodeos/logging.json

```diff
@@ -66,6 +66,41 @@
       "appenders": [
         "stderr"
       ]
+    },{
+      "name": "p2p_log",
+      "level": "info",
```

Comment:
Why is default set to "info"?

Reply:
The default of all loggers is `info`. The previous example `loggin.json` `net_plugin_impl` was set to `info`. So this keeps it the same.